app/views/content/home/_purple-box.html

```diff
@@ -3,15 +3,15 @@
   </div>
 
   <h2 class="steps-to-become-a-teacher__heading">
-    Steps to get into teaching.
+    Steps to become a teacher.
   </h2>
 
   <p class="steps-to-become-a-teacher__text">
-    Find out how to become a qualified teacher in England.
+    Find out what qualifications you need, what funding you can get and how to train to teach.
```

Comment:
The word 'what' being the start of two consecutive things in the list feels a bit funny here. Maybe

```suggestion
    Find out which qualifications you need, what funding you can get and how to train to teach.
```

Reply:
like it